components/nimbus/ios/Nimbus/Nimbus.swift

```diff
@@ -4,6 +4,17 @@
 
 import Foundation
 
+// Depending on build setup, we may be importing Glean as a Swift module
+// or we may be compiled together with it. This detects whether Glean is
+// an external module and makes it available to our entire package if so.
+//
+// Note that the files under `./Utils` are copies of internal files from
+// Glean, and it's very important they they be excluded from any build
+// that is compiling us together with Glean.
+#if canImport(Glean)
+@_exported import Glean
```

Comment:
The use of `@_exported` here means we can avoid having to `import Glean` in every file that wants it, and was the most expedient option. I'm open to the suggestion that we should avoid it given that the leading underscore indicates it is unstable, but on the other hand, firefox-ios imports the existing megazord using `@_exported import MozillaAppServices` so I don't feel too bad about it...

Reply:
I think using the "unstable" API here is fine. I see that used in the wild quite often and if it ever comes to it a stabilization proposal will need to argue real good why behavior should change significantly (at which point we can also still change what we do)